interfaces/builtin/desktop.go

```diff
@@ -266,14 +275,12 @@ func (iface *desktopInterface) MountConnectedPlug(spec *mount.Specification, plu
 		})
 	}
 
-	/*
-		appId := "snap.pkg." + plug.Snap.Name()
```

Comment:
I noticed that the appId has changed, can you explain this? What role does it play?